### ISSUE TITLE
[Fault] Global:RunCommand method doesnt work

### ISSUE THREAD

**Issue by DrKittens** [Niam5/Eluna#1]
Steps to Reproduce:

1. Add this script to lua scripts folder as testaccount.lua
```lua
--testaccount.lua

print("Creating Account example")
RunCommand("account create example password")
```

2. restart mangosd / reload eluna

Expected Behaviour: 
"Creating Account example" is output to the console
eluna successfully uses RunCommand to create an account with username "example"

What Actually Happens:
"Creating Account example" is output to console
No account called "example" is created in DB's accounts table

Suspect Global:RunCommand method isnt firing at all[ due to [line 1228](https://github.com/Niam5/Eluna/blob/eb2fc36270365546824262b027a14153755c7026/GlobalMethods.h#L1228) missing || CMaNGOS

**Comment by Niam5** [comment]
Fixed with 73db0b8067b23de0184354f6e556de4d27c5cdc0

Core changes also completed to add nullptr check around CliHandler::SendSysMessage to prevent crash when a command that gives feedback executes.

**Comment by DrKittens** [comment]
Confirmed fault is resolved, when checking returns for RunCommand("account create example test") it returns "nil" correctly and executes the command sucessfully.

Thank you very much for the quick turnaround!
